CHANGELOG.md

```diff
@@ -130,6 +130,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - Changed `DELETE /pdrs/{pdrname}` API behavior to also delete record from Postgres database.
   - **CUMULUS-2309**
     - Changed `DELETE /granules/{granuleName}` API behavior to also delete record from Postgres database.
+    - Changed `Bulk operation BULK_GRANULE_DELETE` API behavior to also delete records from Postgres database.
+    - Removed `@cumulus/api/models/granule.unpublishAndDeleteGranule` in favor of `@cumulus/api/lib/granule-remove-from-cmr.unpublishGranule` and `@cumulus/api/lib/granule-delete.deleteGranuleAndFiles`.
```

Comment:
This is *probably* technically a breaking change as api/models/* is a public interface.

Reply:
👍 https://github.com/nasa/cumulus/pull/2108/commits/803265db1779356e6df3caf97943af834939085e